File: blotztask-mobile/src/feature/breakdown/models/subtask-dto.ts
import { Timespan } from "react-native/Libraries/Utilities/IPerformanceLogger";

export interface SubtaskDTO {
  taskId: number;
  parentTaskId: number;
  title: string;
  description?: string;
  duration?: Timespan;
  order: number;
  isDone: boolean;
}

File: blotztask-api/Modules/Tasks/DTOs/SubtaskDTO.cs
namespace BlotzTask.Modules.Tasks.DTOs;

public class SubtaskDto
{ 
    public string Title { get; set; } = string.Empty;
    public string? Description { get; set; }
    public TimeSpan? Duration { get; set; }
    public int Order { get; set; }
    public bool IsDone { get; set; } = false; // default for new subtasks
}